### ISSUE TITLE
Add NBT support for items

### ISSUE THREAD

**Issue by Programista1k** [zrdzn/LovelyDrop#35]

**Comment by Programista1k** [comment]
Hey, would it be possible to add custom items support? For example, if you have pickaxe "X" from itemsadder, u are able to drop special additional items from stone.

Also it would be nice to get custom drops too, for example special chests, etc. all items based on custom-nbt.

If its not already implemented, because i haven't digged to that function 0.o

I would be down for paying for plugin with such functionality as "CustomDrops" is hella bugged xD

**Comment by zrdzn** [comment]
Does "ItemsAdder" requires something else other than NBT?

**Comment by Programista1k** [comment]
> Does "ItemsAdder" requires something else other than NBT?

<img width="826" alt="obraz" src="https://user-images.githubusercontent.com/87565210/230217760-0243b909-60a5-44c9-b9f4-07f179c74cf9.png">

Nah, just the custom NBT, by itemsadder integration i mean ability to drag'n drop items to be able to add it fast way :d

**Comment by zrdzn** [comment]
Well all I can do for now is to add NBT key to config, but that may also be a little problematic because on newer version I probably should use PDC instead of pure NBT, I don't know.

**Comment by zrdzn** [comment]
Hello, sorry for long waiting. I've just done some cleanup and added NBT support also. It is available in v2.0 pre-release which will appear in releases tab in few hours or days.

![image](https://github.com/zrdzn/LovelyDrop/assets/87321869/70f65fa1-8e1e-417f-b656-947df54f99ed)
```YAML
  diamondSword:
    item:
      stack:
        material: DIAMOND_SWORD
        item-meta:
          display-name: '&a&lEpic Sword'
          lore:
          - '&6Legendary diamond sword.'
      nbt:
        identifier: sword-drop
        number: 5
        simpleobject:
          wonderifworks: hello
          num: 5651
```

**Comment by zrdzn** [comment]
Alright, I've uploaded latest pre-release build which is available [here](https://github.com/zrdzn/LovelyDrop/releases/tag/v2.0). Feel free to test it out and let me know about any bugs. :smile:
